Stripe/StripeiOSTests/PaymentSheet+APITest.swift

```diff
@@ -218,8 +218,8 @@ class PaymentSheetAPITest: XCTestCase {
         let confirmExpectation = XCTestExpectation(description: "Confirm deferred intent")
         let callbackExpectation = XCTestExpectation(description: "Confirm callback invoked")
 
-        let types = ["ideal", "card", "bancontact", "sofort"]
-        let expected: [STPPaymentMethodType] = [.card, .iDEAL, .bancontact, .sofort]
+        let types = ["card", "cashapp"]
+        let expected: [STPPaymentMethodType] = [.card, .cashApp]
```

Comment:
Maybe we just check if the returned payment methods are not empty?

Reply:
that also makes sense

Reply:
Going to leave as-is for now since we know the root cause